account/README.md

```diff
@@ -1,10 +1,50 @@
 # Account
 
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Installing
+
+In order to get an executable environment for development, some environment variables have to be set first:
+
+| Variable                        | Description                                              | Default Value     |
+|---------------------------------|----------------------------------------------------------|-------------------|
+| PORTFOLIO_URL                   | to send Portfolio-Requests of format: "http://host:port" | localhost:18082   |
+| RABBITMQ_SERVICE                | RabbitMQ service name                                    | localhost         |
+| RABBITMQ_EXCHANGE_STOCKACQUIRED | used for Queue to link Sub with StockAcquiredPublisher   | stock_logs        |
+| RABBITMQ_EXCHANGE_COMMENTTONE   | used for Queue to link Sub with CommentTonePublisher     | comment_tone_logs |
+| RABBITMQ_EXCHANGE_TYPE          | can be one of direct, fanout, topic and headers          | fanout            |
+
+Make sure that the local environment is active. Then the program can be executed using the following command:
+
+```bash
+mvn exec:exec
+```
+
+At the end, you can send a simple REST request to provide feedback for the stock trader application:
+
+```bash
+curl --location --request POST 'localhost:18080/' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "id": "portfolio id",
+    "message": "your message"
+}'
+```
+
+## Documentation
+
+You can get more details about the project with the documented code or the domain stories in the [`doc`][documentation] folder.
+
+[feedback_actions]: https://github.com/whzinformatik/stocktrader/actions?query=workflow%3Aaccount
+[documentation]: ./doc
+
```

Comment:
I would remove the last part of the README because it is part of the code documentation.